docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -140,7 +140,7 @@ You can either bend or soldier the jumper wire here to make the connection betwe
 
 {{% /alert %}}
 
-## Program your plant watering robot
+## Get ready to program your plant watering robot
```

Comment:
I don’t think this change makes sense since adding the code etc. is all within this heading. If you want to make this change, I suggest adding another H2 heading later for actually programming it.

Reply:
i will still add a section with full code so i will apply this comment when i do that change if that's ok with you

Reply:
Okay I won't block for now then. When you do change the H2 headings, I'd suggest "Set up your hardware" for the first one instead of "Set up your plant watering robot," and then this one can become "Set up your software" or something like that, and then "Program your plant watering robot" can come a couple steps down from here.